apiserver/Makefile

```diff
@@ -19,6 +19,7 @@ push-$(1): build-$(1)
 endef
 else
 define DOCKER_IMAGE
+$(3) = gcr.io/$(PROJECT)/$(1):$$(shell docker images -q --no-trunc $(1) | sed -e 's,[^:]*:,,')
```

Comment:
This is non-CI (local) case.  It just uses the local repository and shouldn't define $(3).

Reply:
So if I `make deploy` without this flag, these variables will be missing, and the command will fail